index.js

```diff
@@ -238,7 +265,9 @@ async function run() {
     const roleSessionName = core.getInput('role-session-name', { required: false }) || ROLE_SESSION_NAME;
     const roleSkipSessionTaggingInput = core.getInput('role-skip-session-tagging', { required: false })|| 'false';
     const roleSkipSessionTagging = roleSkipSessionTaggingInput.toLowerCase() === 'true';
-    const webIdentityTokenFile = core.getInput('web-identity-token-file', { required: false })
+    const webIdentityTokenFile = core.getInput('web-identity-token-file', { required: false });
+    let sourceAccountId;
+    let webIdentityToken;
```

Comment:
nit: can we move these declarations above the if-else statement so that it's closer to where they are used